src/handlers/ingress.ts

```diff
@@ -12,10 +12,25 @@ import handlers from './handlers'
 import { isAdmin } from '../util/permissions'
 import { unaliasInvocation } from '../util/unalias'
 import { handleEasterEgg } from './eggs/easter'
+import { addEntry } from '../util/cacheManager';
+import config from '../config/config'
 
 export async function handleMessage(msg: Message) {
-    // Get user or create one if it doesn't exist
     const authorId = msg.author.id
+    const channel = msg.channel.toString()
+
+    if(channel == config.channels.journals){
+        addEntry(msg, 'j')
+        return
+    }
+    else if(channel == config.channels.gratefulness){
+        addEntry(msg, 'g')
+        return
+    }
+
+    if (!shouldRespond(msg)) {return}
```

Comment:
For parallelism, let's expand the `shouldRespond` function inline. So we can add another arm to the `else if` chain and say that `else (channel !== config.channels.progress) // ignore`.